systemd/systemd.go

```diff
@@ -614,14 +623,45 @@ func (s *systemd) getUnitStatus(properties []string, unitNames []string) ([]*Uni
 				}
 			}
 			if len(missing) > 0 {
-				return nil, fmt.Errorf("cannot get unit %q status: missing %s in ‘systemctl show’ output", cur.UnitName, strings.Join(missing, ", "))
+				return nil, fmt.Errorf("cannot get unit %q status: missing %s in ‘systemctl show’ output", cur.Id, strings.Join(missing, ", "))
```

Comment:
conceptually we want cur.Name here, so my remark about setting it early

Reply:
Thank you. I replaced two cur.Id early references with the correct cur.Name. 
Fixed.